### ISSUE TITLE
[feat] open a good first issue

### ISSUE THREAD

**Issue by gogo2464** [keep-starknet-strange/blobstream-starknet#145]
Hi I would like to contribute more.

My issue where I contribute is currently stucked by another PR.

I want to avoid rebase conflict. could you create and assign me to a good first issue please?

**Comment by ShantelPeters** [comment]
I am applying to this issue via [OnlyDust platform](https://app.onlydust.com/p/blobstream-starknet).

### My background and how it can be leveraged
Please can I be assigned to this issue 

### How I plan on tackling this issue
To approach this problem, I would:

1. **Communicate Clearly**: Reach out to the maintainers, expressing my interest in contributing more and explaining the situation with the current PR.
2. **Request Assignment**: Politely request a new, beginner-friendly issue that I can work on while the current PR is being resolved.
3. **Monitor Progress**: Keep an eye on the current PR to rebase and address any conflicts as soon as it's merged.

**Comment by gogo2464** [comment]
I would love to contribute again. Could you let me just finish a very hard PR on another PR before contributing again please?

**Comment by tmanas06** [comment]
I am applying to this issue via [OnlyDust platform](https://app.onlydust.com/p/blobstream-starknet).

### My background and how it can be leveraged
first Find an existing "good first issue":

If you're contributing to an open-source project on platforms like GitHub, most repositories have a "good first issue" label to help contributors find beginner-friendly tasks. You can search for these issues by filtering through the repository's issue tracker and applying the "good first issue" label.

Create a new "good first issue":

If you have sufficient permissions on the project, you can create a new issue that qualifies as a "good first issue." This should be a smaller, self-contained task, like fixing documentation, improving test coverage, or addressing minor bugs.

Avoid rebase conflicts:

Work on the new issue in a separate branch.
Once your original PR is unblocked, you can merge the latest changes into your branch to minimize potential conflicts.

### How I plan on tackling this issue
I usually follow the above steps on my organisation projects, hope it finds you informative...

**Comment by SumitGoulikar** [comment]
I am applying to this issue via [OnlyDust platform](https://app.onlydust.com/p/blobstream-starknet).

### My background and how it can be leveraged
I'm an undergraduate in Computer Science and am well experienced in coding across different programming languages. Along the way, I have enjoyed building efficient algorithms, practicing data structures, and tackling problems in clean, optimized code. So far, I've had fun using best practices to enhance performance and readability in projects.

I'm excited about this project because it will take my coding abilities, problem-solving skills, and passion for learning to the benefit of improving and maintaining code quality. With my strong foundation in computer science and eagerness to collaborate on impactful projects, you can expect to receive valuable contributions from me.

**Comment by gogo2464** [comment]
Hello.

I am looking for a good first issue in cryptography. I am not very strong in cryptography but it is a field where I would like to learn. I am strong in coding but I would LOVE doing cryptogrtaphy.

**Comment by ShantelPeters** [comment]
I am applying to this issue via [OnlyDust platform](https://app.onlydust.com/p/blobstream-starknet).

### My background and how it can be leveraged
i am a frontend and blockachain developer with a strong background in cario also have experience with html, css, react, solidity, javascript,typescript . 

### How I plan on tackling this issue
Request a "good first issue" from the project maintainers and ask to be assigned, explaining your current contribution is blocked by another PR and you'd like to avoid rebase conflicts.

**Comment by sun-flame** [comment]
I am applying to this issue via OnlyDust platform.

**My background and how it can be leveraged**

I am a MERN stack developer with expertise in MongoDB, Express.js, React, and Node.js. I also have experience with JavaScript, TypeScript, and frontend development tools, making me well-versed in building and debugging efficient, scalable applications. My technical background enables me to approach this task with a strong understanding of modern development practices and low-level debugging concepts.

**How I plan on tackling this issue**

I plan to request a "good first issue" from the project maintainers and ask to be assigned. This approach ensures I can get started on a manageable task and gradually familiarize myself with the project's codebase. I will also communicate openly with maintainers, explaining if my current contribution is blocked by another PR, to avoid potential rebase conflicts. I aim to implement the solution in a systematic manner, ensuring all changes are tested and aligned with the project's standards.

**Comment by ryzen-xp** [comment]
Can I try solving this issue?

**Comment by chiscookeke11** [comment]
Can I work on this, please?

**Comment by chiscookeke11** [comment]
Is it okay if I tackle this?

**Comment by Verifieddanny** [comment]
I'd like to handle this task.

**Comment by yashh1994** [comment]
i will do this for minimum deal.

**Comment by mimisavage** [comment]
Mind if I try this one?
